### PR TITLE
Fix defects in timer on tap app

### PR DESCRIPTION
**Pull request by mani-vandrangi**
Fixed the following issues in timer_on_tap app

1. Updated README.md to add configs before testing
2. Fixed errors in FDK validate
3. Fixed bootstrap jquery dependency error
4. Fixed agents not showing error

### CODE REVIEW

File: Freshworks-Samples/Freshdesk/timer_on_tap/app/startTimer.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <script src="https://static.freshdev.io/fdk/2.0/assets/fresh_client.js"></script>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/Freshdesk/timer_on_tap/app/startTimer.html:6]: Please remove the jQuery as it is not used anywhere. If used anywhere, please replace the function with Vanilla JS.

**Review by ravirajsubramanian — CHANGES_REQUESTED** [review]

File: Freshworks-Samples/Freshdesk/timer_on_tap/app/startTimer.html (hunk 1 above)
Comment (ravirajsubramanian) [reply Freshworks-Samples/Freshdesk/timer_on_tap/app/startTimer.html:6]: This has been added as a separate issue #272. So, merging this PR.

**Review by ravirajsubramanian — APPROVED** [review]

**Review by ravirajsubramanian — APPROVED** [review]

**Review by Saif-Shines — APPROVED** [review]

**Comment by mani-vandrangi** [comment]
Abandoning my PR as changes are already merged in another PR

**Comment by ravirajsubramanian** [comment]
These changes have been fixed in the PR #275.

**Pull request closed without merging**